docs/src/main/asciidoc/security-openid-connect-client.adoc

```diff
@@ -369,8 +369,8 @@ You will be asked to log in into a `Single Page Application` provided by `OpenID
  ** accessing `/frontend/user-name-with-propagated-token` will return `200`
  ** accessing `/frontend/admin-name-with-propagated-token` will return `403`
  * Logout and login as `admin` (password: `admin`) who has both `admin` and `user` roles
- ** accessing `/frontend/user-name-with-propagated-token` will return `200`
- ** accessing `/frontend/admin-name-with-propagated-token` will return `200`
+ ** accessing `/frontend/user-name-with-propagated-token` will return `403`
```

Comment:
@skraft-redhat Given that the admin user has both `admin` and `user` roles, `200` is expected, do you mean the demo does not work correctly ?